src/test/end-to-end/helpers/PuppeteerHelper.js

```diff
@@ -30,7 +30,7 @@ class PuppeteerHelper extends Helper {
         if (testConfig.TestForXUI) {
             const tabXPath = `//div[text()='${tabTitle}']`;
             const clickableTab = await helper.page.$x(tabXPath);
-            await clickableTab[0].click();
+            await helper.page.evaluate(el => el.click(), clickableTab[0]);
```

Comment:
Did we try await I.waitForClickable here then a click?

Reply:
The clickableTab is element handle fetch from xpath. so i have updated method to use waitForXpath